bitnami/grafana/templates/deployment.yaml

```
@@ -79,7 +79,12 @@ spec:
           image: {{ include "volumePermissions.image" . }}
           imagePullPolicy: {{ .Values.volumePermissions.image.pullPolicy | quote }}
           command:
-            - chown -R {{ .Values.containerSecurityContext.runAsUser }}:{{ .Values.podSecurityContext.fsGroup }} /bitnami/grafana
+            - /bin/bash
+            - '-ec'
+          args:
+            - |
+              mkdir -p /bitnami/grafana
+              chown -R {{ .Values.grafana.containerSecurityContext.runAsUser }}:{{ .Values.grafana.podSecurityContext.fsGroup }} /bitnami/grafana
```

We should skip some hidden files that comes with the mounted volumes:
```suggestion
              find /bitnami/grafana -mindepth 1 -maxdepth 1 -not -name ".snapshot" -not -name "lost+found" | xargs chown -R {{ .Values.grafana.containerSecurityContext.runAsUser}}:{{ .Values.grafana.podSecurityContext.fsGroup }}
```